com.unity.netcode.gameobjects/Runtime/Configuration/NetworkConfig.cs

```diff
@@ -224,7 +224,7 @@ public ulong GetConfig(bool cache = true)
                 return m_ConfigHash.Value;
             }
 
-            var writer = new FastBufferWriter(MessagingSystem.NON_FRAGMENTED_MESSAGE_MAX_SIZE, Allocator.Temp);
+            var writer = new FastBufferWriter(MessagingSystem.NON_FRAGMENTED_MESSAGE_MAX_SIZE, Allocator.Temp, int.MaxValue);
```

Comment:
Perhaps int.MaxValue >> 8?
Just wondering if allowing for a 2GB buffer to send is a good idea?

Reply:
Just started looking at your other PRs and now better understand this change.